css/cbf-programs-style.css

```diff
@@ -295,189 +295,115 @@ p.header-explore {
 }
 
 .legend-title {
-	position:relative;
-	bottom:0;
-	right:0;
-    text-align: left;
-    margin-bottom: 5px;
+  margin-bottom: 5px;
 	font-family: "Work Sans", helvetica, arial, sans-serif;
-    font-weight: 700;
+  font-weight: 700;
 	color:#2C2A29;
-    font-size: 90%;
+  font-size: 90%;
 	padding-bottom: 10px;
-    }
-  .legend .legend-scale ul {
-    margin: 0;
-    margin-bottom: 5px;
-    font-family: "Work Sans", helvetica, arial, sans-serif;
-    font-weight: 500;
-	padding: 0;
-    float: left;
-    list-style: none;
-    }
-  .legend .legend-scale ul li {
-    font-size: 13px;
-    list-style: none;
-    margin-left: 0;
-    line-height: 20px;
-    margin-bottom: 2px;
-    }
-  .legend ul.legend-labels li span {
-    display: block;
-    float: left;
-    height: 14px;
-    width: 24px;
-    margin-right: 5px;
-    margin-left: 0px;
-	margin-top: 3px;
-    border: 0px solid #999;
-    }
-  .legend ul.legend-labels li ul { /* to indent nested lists in the legend */
-  	margin-left: 1em;
+}
+
+.legend .legend-scale ul {
+  margin: 0;
+  margin-bottom: 5px;
+  font-family: "Work Sans", helvetica, arial, sans-serif;
+  font-weight: 500;
+  padding: 0;
+  list-style: none;
   }
+.legend .legend-scale ul li {
+  display: flex;
+  font-size: 13px;
+  line-height: 20px;
+  margin-bottom: 10px;
+  }
+.legend ul.legend-labels li span {
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
I am guessing that you had varying widths for these so that the backgrounds didn't repeat and look all funky for the box-shaped ones? But it made the text alignment hard to get right, so I made them all the same width and set the background to not repeat.

Reply:
> I am guessing that you had varying widths for these so that the backgrounds didn't repeat and look all funky for the box-shaped ones? But it made the text alignment hard to get right, so I made them all the same width and set the background to not repeat.

Haha YES you are correct, thank you for doing it properly